requirements/base.txt

```diff
@@ -4,7 +4,7 @@ natsort>=6.0.0
 prettytable
 protobuf>=3.20.0
 pyyaml
-datumaro==1.2.0rc3
+datumaro@ git+https://github.com/openvinotoolkit/datumaro@e9e5307efdbd51598cbfc9fe3fd7c282870b98ef#egg=datumaro
```

Comment:
Should we enforce latest branch for datumaro?

Reply:
I don't see a reason for that, unless we need to use the latest features. Setting develop here would lead to rolling upgrade of datumaro and may cause unpredictable effect in tests.
Prior to OTX release, this requirement should be changed to a package version (which doesn't exist yet).